newrelic.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['AboutImages'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: 'd55311fa5001947bf8905d3fda08200af2a68211',
```

Comment:
your API key should be in `process.env` or your `newrelic.js` should be `.gitignore`d